src/main/java/net/minestom/server/instance/block/BlockSoundImpl.java

```diff
@@ -0,0 +1,23 @@
+package net.minestom.server.instance.block;
+
+import net.minestom.server.registry.Registry;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.Collection;
+
+public record BlockSoundImpl(Registry.BlockSoundTypeEntry registry) implements BlockSoundType {
+    private static final Registry.Container<BlockSoundType> CONTAINER = Registry.createStaticContainer(Registry.Resource.BLOCK_SOUND_TYPES,
+            (namespace, properties) -> new BlockSoundImpl(Registry.blockSoundTypeEntry(namespace, properties)));
```

Comment:
Is there not the generated interfaces missing?

Reply:
No? What are you expecting here?

Reply:
I could do that, but the elements are attached to the Block Registry Entries and my intent is that you access them through the blocks.
I can have them standalone like the Material list if that's requested.

Reply:
Talked with mattw, added the list.